### PR TITLE
feat: update shell competion and version match test

### PR DESCRIPTION
**Pull request by mrinalwadhwa**
wait to merge until we have a release binary that supports `ockam completion`

### CODE REVIEW

File: .github/ockam.rb.template [hunk 1]
```diff
@@ -24,6 +27,9 @@ class Ockam < Formula
       def install
         mv "ockam.x86_64-apple-darwin", "ockam"
         bin.install "ockam"
+        (bash_completion/"ockam").write `#{bin}/ockam completion --shell bash`
+        (fish_completion/"ockam.fish").write `#{bin}/ockam completion --shell fish`
+        (zsh_completion/"_ockam").write `#{bin}/gh ockam --shell zsh`
```
Comment (metaclips) [inline .github/ockam.rb.template:32]: Is `#{bin}/gh ockam` intended?

**Review by metaclips — APPROVED** [review]
I believe we can merge this now. ockam.rb will be updated on next released

File: .github/ockam.rb.template (hunk 1 above)
Comment (mrinalwadhwa) [reply .github/ockam.rb.template:32]: thanks for catching that.

Comment (mrinalwadhwa) [reply .github/ockam.rb.template]: fixed.

**Pull request merged**